test/specs/registration/CellPhoneNumberDesign.js

```diff
@@ -0,0 +1,178 @@
+import { assert } from 'chai';
+import { url } from '../examples/constants';
+
+const inputFieldPhone = '//input[@name="phone"]';
+
+describe('Cell Phone Number input field', () => {
```

Comment:
Change to the more precise title for describe

Reply:
Done